### ISSUE TITLE
Custom font bug

### ISSUE THREAD

**Issue by Wector11211** [sp614x/optifine#1192]
Hello. It seems that I've found a bug. Custom font selection with optifine works really bad, but without it - good.
Do you know how to fix that with optifine?
https://streamable.com/03lfe - optifine
https://streamable.com/a7end - no optifine

**Comment by sp614x** [comment]
Resource pack link or font texture?

**Comment by Wector11211** [comment]
I don't think it is because of resource pack. I tried several custom fonts but result was the same.
[minecraft font.zip](https://github.com/sp614x/optifine/files/1690393/minecraft.font.zip)
